scripts/zones/Nyzul_Isle/mobs/Stealth_Bomber_Gagaroon.lua

```diff
@@ -0,0 +1,104 @@
+-----------------------------------
+--  MOB: Stealth Bomber Gagaroon
+-- Area: Nyzul Isle
+-- Info: Enemy Leader, Thief job, runs around dropping bombs
+-----------------------------------
+require("scripts/globals/pathfind")
+require("scripts/globals/status")
+require("scripts/globals/nyzul")
+-----------------------------------
+local entity = {}
+
+local function pickRunPoint(mob)
+    mob:setLocalVar("ignore", 1)
+    local distance   = math.random(10, 25)
+    local angle      = math.random() * math.pi
+    local fromTarget = mob:getTarget()
+
+    if fromTarget == nil then
+        fromTarget = mob
+    end
+
+    local pos = GetFurthestValidPosition(fromTarget, distance, angle)
```

Comment:
Just making a note that this function does not appear to exist in our current codebase.